.github/workflows/upgrade-test.yml

```diff
@@ -247,6 +246,24 @@ jobs:
           path: |
             /tmp/chainflip/*/chainflip-*.*log
 
+      - name: Upload Chainflip Logs 💾
+        if: always()
+        continue-on-error: true
+        uses: actions/upload-artifact@v3
+        with:
+          name: chainflip-logs
+          path: |
+            /tmp/chainflip/logs/*.log
```

Comment:
The execWithLog utilities write to the directory by default, so any logs added here will be uploaded.